ictv/pages/local_login.py

```diff
@@ -46,6 +46,7 @@ def POST(self):
             user = User.selectBy(email=form.email, password=hash_password(form.password)).getOne(None)
             if not user:
                 raise ImmediateFeedback('login', 'invalid_credentials')
+            user.last_connection = datetime.datetime.now()
```

Comment:
This can be set here https://github.com/UCL-INGI/ICTV/blob/master/ictv/app.py#L215 I believe, it is more general and scalable